lib/model/content.dart

```diff
@@ -1081,7 +1094,16 @@ class _ZulipContentParser {
       items.add(parseImplicitParagraphBlockContentList(item.nodes));
     }
 
-    return ListNode(listStyle!, items, debugHtmlNode: debugHtmlNode);
+    if (listStyle == ListStyle.ordered) {
+      final startAttr = element.attributes['start'];
+      final start = startAttr != null
+        ? int.tryParse(startAttr, radix: 10)
+        : 1;
```

Comment:
nit:
```suggestion
      final start = startAttr == null ? 1
        : int.tryParse(startAttr, radix: 10);
```

That puts the simpler case first, getting it out of the way.